### PR TITLE
Add bloodland weapons that were created in c# but are not accessible

### PR DESCRIPTION
**Pull request by Chilly021**

### CODE REVIEW

**Review by zrikzlok — APPROVED** [review]

**Review by Leolus — APPROVED** [review]

**Review by sproatba — APPROVED** [review]

**Comment by Chilly021** [comment]
Waiting until tomorrow to merge. Need a day to make sure not too many things are in flight.

**Pull request merged**
